$NetBSD: patch-ab,v 1.3 2015/12/29 23:34:56 dholland Exp $

Configure for pkgsrc.
This is not the only scheme interpreter; name the program accordingly.

--- Makefile.orig	Sun Jan 21 22:15:47 1996
+++ Makefile
@@ -1,6 +1,6 @@
 # Makefile for the UMB Scheme interpreter.
-CC = gcc
-CFLAGS =  -g -O   -pedantic -Wall
+#CC = gcc
+#CFLAGS =  -g -O   -pedantic -Wall
 
 #CC = cc
 #CFLAGS = -O 
@@ -11,7 +11,7 @@
 	  io.o compiler.o eval.o architecture.o number.o\
 	  fixnum.o bignum.o rational.o real.o complex.o
 libraries = -lm
-program = scheme
+program = umb-scheme
 
 .PHONY:		default tags print
 
